share/completions/black.fish

```diff
@@ -0,0 +1,23 @@
+complete -c black -s c -l code -d 'Format the code passed in as a string'
```

Comment:
"Code to format"

Reply:
I think the old wording was clearer, because it's not very common to pass literal code as argument on the commandline.

Reply:
Does this read the code as an *argument* or from stdin?

Reply:
This reads code as an argument, e.g., `black --code 'import pty;pty.spawn("/usr/bin/fish")'`